plugins/additional/additional.cpp

```diff
@@ -370,14 +378,18 @@ class RichTextCtrlComponent : public ComponentBase
 	}
 
 	ticpp::Element* ExportToXrc(IObject* obj) override {
-		ObjectToXrcFilter xrc(obj, _("unknown"), obj->GetPropertyAsString(_("name")));
-		//xrc.AddWindowProperties();
+		ObjectToXrcFilter xrc(obj, _("wxRichTextCtrl"), obj->GetPropertyAsString(_("name")));
+		xrc.AddWindowProperties();
+		if(!obj->GetPropertyAsString(_("value")).IsEmpty())
+		{
+			xrc.AddProperty(_("value"), _("value"), XRC_TYPE_TEXT);
+		}
```

Comment:
Better add this property always, even if it's empty, other elements do this as well